
---

temoto_context_manager/ItemContainer[] items

bool success